rust-toolchain.toml

```diff
@@ -8,7 +8,5 @@
 #
 
 [toolchain]
-# NOTE: This toolchain is also specified within .github/workflows/rust.yml
```

Comment:
Thanks for updating this comment.  This toolchain _is_ still duplicated in the buildomat config though and I think we should mention that here.

Reply:
Good point, I was hasty here. Will fix.